tools/quicktype-wrapper/README.md

```diff
@@ -2,16 +2,37 @@
 
 This node module that wraps the `quicktype` tool to generate code using JSON Schema.
 
-## Build and Run
+## Install
+
+This tool can be installed as a global CLI or run as a Node script. Requires Node 8+.
+
+### CLI (Recommended)
+
+Install the CLI globally:
 
 ```sh
-npm run build
-npm start
+sudo npm run cli
 ```
 
-This builds the script in the `build/` directory. Then runs the script.
+Then type `qt` to run the generator.
+
+### Node Script
+
+You can also run as a script. Build then run:
+
+```sh
+npm run compile
+npm run start
+```
+
+## Run
+
+const IN = process.env.IN || `${HOMEDIR}/Documents/github/googleapis/google-cloudevents/proto/`;
+const OUT = process.env.OUT || `${HOMEDIR}/Documents/out`;
```

Comment:
Working in Documents seems odd? I assume github/googleapis/google-cloudevents/proto is coming from a separately git cloned repository?

Reply:
These are some default values that if you don't specify one.
(I recommend we always specify one)

It's nice being able to just execute `qt`.

We could consider not having defaults or just writing an error, but working in `Documents` rather than cluttering ~ is better imo.

Reply:
Is it normal in Node to use environment variables like this, rather than providing them as command-line arguments? In other environments I'd *definitely* be using command line arguments, and I wouldn't assume any defaults - it feels to me like the chances of the defaults being right for more than a single person are pretty small.

Reply:
The goal of this tool is not to build a CLI to generate, it's to use the `npm run gen` script.

I think I can just remove these defaults.

Reply:
Removed defaults.